manifests/05-gcp-ro-credentialsrequest.yaml

```diff
@@ -17,3 +17,6 @@ spec:
   secretRef:
     name: cloud-credential-operator-gcp-ro-creds
     namespace: openshift-cloud-credential-operator
+  serviceAccountNames:
+  - cloud-credential-operator
```

Comment:
I'm assuming this is more of an example of the kinds of changes that need to be made across the various repos, and CCO - as presently written - will not actually make use of these creds when in Manual mode. Correct?

Reply:
thats right